controller-server/src/main/java/com/yahoo/vespa/hosted/controller/versions/VersionStatus.java

```diff
@@ -120,8 +120,9 @@ public static VersionStatus compute(Controller controller, Version currentVersio
 
     private static ListMap<Version, String> findConfigServerVersions(Controller controller) {
         List<URI> configServers = controller.zoneRegistry().zones().stream()
+                // TODO: Filter properly.
                 .filter(zone -> ! zone.region().equals(RegionName.from("us-east-2a")))
-                .flatMap(zone -> controller.getConfigServerUris(zone.environment(), zone.region()).stream())
+                .flatMap(zone -> controller.getConfigServerUris(zone).stream())
```

Comment:
Consider renaming the lambda variable `zone` to `zoneId`.

Reply:
Yes, there may be a lot more of these lying around. I didn't prioritise looking for them, but I will take a look.

I'll rename this in my next PR.